Create Table Scripts.txt

```diff
@@ -1,78 +1,78 @@
-CREATE DATABASE Project;  
+CREATE DATABASE Project;
 USE Project;
 
 CREATE TABLE Dentist (
-    DentistId INT PRIMARY KEY,
-    PhoneNumber VARCHAR(15) NOT NULL,
+    Dentist_Id INT PRIMARY KEY,

```

Comment:
I think it's better to be consistent in naming tables and columns. I prefer to use snake_case instead of CamelCase because SQL is case-insensitive. I know there is no definitive rule though.